### PR TITLE
Adding LIS in kotlin

### PR DESCRIPTION
**Pull request by shubhammantri1**
Fixes #2253 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Adding Longest_Increasing_Subsequence in Kotlin

#### Languages Used:
- Kotlin

#### Files Added:
- Longest_Increasing_Subsequence.kt

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by shubhammantri1** [comment]
@jainaman224 @SakshiShreya Please Review my PR.

**Comment by shubhammantri1** [comment]
@jainaman224 @SakshiShreya @rishabhgarg25699 I think my PR's are not visible to you. I think there is a glitch in Github.

File: Longest_Increasing_Subsequence/Longest_Increasing_Subsequence.kt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import java.util.*
+
+fun main() {
```
Comment (asha15) [inline Longest_Increasing_Subsequence/Longest_Increasing_Subsequence.kt:3]: keep a extra line

File: Longest_Increasing_Subsequence/Longest_Increasing_Subsequence.kt [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    // Taking Inputs
+    var read = Scanner(System.`in`)
+    println("Enter the size of Array:")
+    val arrSize = read.nextLine().toInt()
+    var arr = IntArray(arrSize)
+    println("Enter the elements")
```
Comment (asha15) [inline Longest_Increasing_Subsequence/Longest_Increasing_Subsequence.kt:9]: keep a extra line

File: Longest_Increasing_Subsequence/Longest_Increasing_Subsequence.kt [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    for(i in 0 until arrSize)
+        arr[i] =  read.nextLine().toInt()
+    var list = IntArray(arrSize)
```
Comment (asha15) [inline Longest_Increasing_Subsequence/Longest_Increasing_Subsequence.kt:12]: keep a line space after this

File: Longest_Increasing_Subsequence/Longest_Increasing_Subsequence.kt [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+ 5
+Enter the elements
+ 5
+ 4
+ 3
+ 2
+ 5
+Size of the longest increasing subsequence: 2
+*/
+
```
Comment (asha15) [inline Longest_Increasing_Subsequence/Longest_Increasing_Subsequence.kt:51]: remove this line

**Comment by shubhammantri1** [comment]
I am sorry @asha15 I am not contributing for now. Please assign this to anyone else.

**Pull request closed without merging**